client/hosting/sites/components/dotcom-style.scss

```diff
@@ -473,6 +473,7 @@
 			margin-top: 0;
 			margin-left: 0;
 			margin-right: 0;
+			margin-bottom: auto;
```

Comment:
This fixes the spacing at the bottom of the tabs.

Without this, it wasn't respecting the padding.

<img width="1143" alt="Screenshot 2024-10-18 at 18 32 26" src="https://github.com/user-attachments/assets/7a788ed4-a5a1-4f4a-b2d6-28120e2debc6">